src/test/java/fi/otavanopisto/kuntaapi/test/AbstractIntegrationTest.java

```diff
@@ -814,6 +828,15 @@ private void setLog4jLevel(Level level) {
     loggerContext.updateLoggers();
   }
   
+  protected void waitMs(int ms) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)